deploy-as-code/helm/environments/dev.yaml

```diff
@@ -270,11 +270,12 @@ inbox:
+"BIRTH":{"totalBirthApplication":"number","totalCollection":"number","totalDownloadsByMobileApp":"number","totalDownloadsByWeb":"number","totalDownloads":"number","delayedRegistration":"number"}}'
```

Comment:
Can you change totalBirthApplication to totalBirthApplications? Seems more intuitive that way.